osu.Game.Rulesets.Tau/Mods/TauModFlashlight.cs

```diff
@@ -26,7 +26,7 @@ namespace osu.Game.Rulesets.Tau.Mods
 {
     public class TauModFlashlight : TauModFlashlight<TauHitObject>
     {
-        public override double ScoreMultiplier => 1.12;
+        public override double ScoreMultiplier => 1.2;
```

Comment:
I don't know how I feel about this change, while sure the FL mod may be harder than std's, I don't think it's by this much. Maybe a value of `1.16` would be better?